sources/platform/actors/development/actor_definition/source_code/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Source code
+description: Learn about Actor source code structure, metamorph feature, or how to run web server inside of an Actor.
+slug: /actors/development/actor-definition/source-code
+---
+
+**Learn about Actor source code structure, metamorph feature, or how to run web server inside of an Actor.**
+
+---
+
+Apify actor's source code placement is defined by its [Dockerfile](../dockerfile.md). If you have created the actor from one of Apify's [templates](https://apify.com/templates) then it's by convention placed in the `/src` directory.
+
+It's completely up to you what language and technologies, including various binaries (Chrome browser, Selenium, Cypress, or any other dependency of your choice) you use in your project. The only requirement is that you have to define the Dockerfile that will build the image for your actor, including all the dependencies and your source code.
+
+In the following chapter, we take a look at the **[example setup of Actor's Dockerfile and source code](./example_setup.md)** and then take a look at two important features supported by the Apify platform:
+
+import Card from "@site/src/components/Card";
+import CardGrid from "@site/src/components/CardGrid";
+
+<CardGrid>
```

Comment:
I find it weird to have two such edge-case use cases promoted with big cards on the source code index. It makes them look as the most important thing that everybody should know about.
<img width="927" alt="CleanShot 2023-04-12 at 17 33 03@2x" src="https://user-images.githubusercontent.com/23726914/231508072-48c52d62-43be-475e-a92f-f0a13dae0da3.png">

Reply:
For example, nowhere under Source code we talk about the differences between Web IDE, git repository integration and all the other important source code topics.